CHANGELOG.md

```diff
@@ -1,4 +1,55 @@
-# Changelog
+# Voila Changelog
+
+## 0.5.0 - Highlights
+
+Below are the major highlights in Voila 0.5.0
+
+### New JupyterLab-based frontend.
+
+In 0.5.0, the frontend of Voila is rebuilt from scratch with JupyterLab 4.0 components. The current `requirejs` + `nbextensions` approach is replaced by a JupyterLab plugin-based application.
+This allows Voila to profit from all the upstream features and improvements.
+
+### New extension loading system.
+
+This is a direct benefit of the new frontend, now Voila will use the same federated extension system of JupyterLab.
```

Comment:
The JupyterLab docs usually refer to them as "prebuilt", so maybe we can use the same word here:

```suggestion
This is a direct benefit of the new frontend, now Voila will use the same prebuilt extension system as in JupyterLab.
```